app/controllers/api/v1/users_controller.rb

```diff
@@ -0,0 +1,8 @@
+class Api::V1::UsersController < ApplicationController
+  # before_action :authenticate_user!
+
```

Comment:
Please implement the API endpoint for user `Registration` as it was requested by the previous code reviewer.

Reply:
Hi @SajjadAhmad14 please is that the requirement in this task?

Reply:
Hello @kingqabil, yes it is a requirement you are supposed to implement user registration endpoints

Reply:
![image](https://user-images.githubusercontent.com/24355301/164891131-5ff7a90b-23b4-400d-b577-ba842b39d2e4.png)
no it's not requirement did you read requirement ?